dockershrink/generator/generator.py

```diff
@@ -0,0 +1,16 @@
+from typing import Dict, Optional
+from dockershrink.ai import AIService
+from dockershrink.package_json import PackageJSON
+
+
+class Generator:
+    def __init__(self, ai_service: Optional[AIService] = None):
+        self.ai = ai_service
+
+    def generate_docker_files(self, analysis: PackageJSON) -> Dict[str, str]:
```

Comment:
I think this should be `analysis: dict`. The analysis object that PackageJSON.analyze() returns is a dict.
Correct me if I'm wrong.